src/components/OrganizationSaas/Dashboard/Create.tsx

```diff
@@ -1,24 +1,167 @@
-import { Button } from '@vocdoni/chakra-components'
-import { useClient } from '@vocdoni/react-providers'
-import { useTranslation } from 'react-i18next'
-import { AccountCreate } from '~components/Account/Create'
-import { ContentsBox } from './Box'
+import { Box, Button, Flex, Grid, Heading, Image, Link, List, ListItem, Text, UnorderedList } from '@chakra-ui/react'
+import { Trans } from 'react-i18next'
+import { AccountCreate } from '~src/themes/saas/components/Account/Create'
+import DarkModeToggle from '~src/themes/saas/components/DarkMode'
+import Wrapper from '~src/themes/saas/components/wrapper'
+import barca from '/assets/barca.png'
+import bellpuig from '/assets/bellpuig.svg.png'
+import berga from '/assets/berga.svg.png'
+import bisbal from '/assets/bisbal.svg'
+import bloock from '/assets/bloock.png'
+import coec from '/assets/coec.png'
+import decidim from '/assets/decidim.png'
+import erc from '/assets/erc.svg'
+import omnium from '/assets/omnium.png'
+import ticanoia from '/assets/ticanoia.png'
 
-const CreateOrganization = () => {
-  const { t } = useTranslation()
+const CreateOrganization = () => (
+  <Wrapper flexDirection={{ base: 'column', lg: 'row' }} px='0' position='relative'>
+    <Box
+      flex='1 1 50%'
+      py={{
+        base: '20px',
+        sm: '40px',
+      }}
+      px={{
+        base: '10px',
+        sm: '20px',
+      }}
+    >
+      <Box maxW='600px' mx='auto'>
+        <AccountCreate />
+      </Box>
+    </Box>
+    <Box
+      position={{ lg: 'relative' }}
+      display='flex'
+      flexDirection='column'
+      flex='1 1 50%'
+      py={{
+        base: '20px',
+        sm: '40px',
+      }}
+      px={{
+        base: '10px',
+        sm: '20px',
+      }}
+      color='white'
+      className='brand-gradient'
+      borderBottomLeftRadius={{ lg: '120px', xl: '200px' }}
+    >
+      <DarkModeToggle position='absolute' ml='auto' top='10px' right='10px' />
+      <Box display='flex' flexDirection='column' maxW='600px' mx='auto' flexGrow={1}>
+        <Heading textAlign='center' mb='48px'>
+          <Trans>Try Vocdoni for free for 7 days</Trans>
+        </Heading>
+        <Box width='fit-content' mx='auto' mb='48px' fontWeight='500'>
+          <UnorderedList>
+            <li>Full access to basic features</li>
+            <li>Unlimited creation of voting processes</li>
+            <li>Multiple administrators</li>
+            <li>Up to 20 voters</li>
+            <li>Support during the trial period</li>
```

Comment:
Should use `ListItem` instead of `li`? https://v2.chakra-ui.com/docs/components/list/usage#unordered-list